backoffice/src/Views/Account/Index.cshtml

```diff
@@ -20,20 +20,52 @@
         </div>
     </div>
     
-    <div class="row mt-4">
-        <div class="col">
-            <div class="card">
-                <div class="card-header">
-                    <h4 class="mb-0">My Assigned Droits</h4>
+    <ul class="nav nav-tabs" id="droitTabs" role="tablist">

```

Comment:
Probably better to have these in cards one above the other rather than in tabs

Tabs means they have to click to check if they have any, rather that them just being displayed to them